pangea-sdk/service/redact/api.go

```diff
@@ -94,6 +94,9 @@ type TextOutput struct {
 	// The redacted text.
 	RedactedText *string `json:"redacted_text"`
 
+	// Number of redactions present in the response
+	Count int
```

Comment:
Is there a reason to not include the json struct tag here?

```suggestion
	Count int `json:"count"`
```